src/lib/receiver.js

```diff
@@ -281,9 +289,13 @@ function createReceiver (opts) {
     }
 
     const fulfillment = cc.toFulfillmentUri(conditionPreimage)
-    debug('about to submit fulfillment: ' + fulfillment)
+
     // returning the promise is only so the result is picked up by the tests' emitAsync
-    return client.fulfillCondition(transfer.id, fulfillment)
+    return reviewPayment(paymentRequest)
```

Comment:
If `reviewPayment()` rejects, the `catch` will print `'error submitting fulfillment', err`, which isn't accurate in this case.